website/frontend/src/helpers.ts

```diff
@@ -57,3 +60,13 @@ export const signPayload = async (injector: any, address: string, payload: any,
     console.error(err);
   }
 };
+
```

Comment:
```suggestion
const isDevChain = () => localStorage.getItem('chain') === 'Development';
```
Add isDevelopmentChain() function to reuse it across components instead of `const chain = localStorage.getItem('chain');` and `if (chain === 'Development')`